docs/arch/adr-03.md

```diff
@@ -0,0 +1,86 @@
+# ADR 03: Azure Data Storage Tools
+
+* Status: Awaiting Review
+* Date: Dec 23, 2019
+
+## Context
+
+When a transcription is approved, a set of flat files containing the transcription data will be saved to Azure. Users will have the option to download a zip file containing their requested subject, group, workflow, or project. Depending on the speed at which we are able to zip the necessary files, we will either trigger a direct download, or provide a link to the location of the zip file to the user. 
+
+The goal is to investigate Azure’s storage options (specifically Blob Storage and File Services) and decide which tool is best suited for our needs.
+
+### Factors to consider:
+
+* How easy is it to share a file to the end user? What is the process for this?
+* Ease of use, how complicated is it to set up, maintain, edit
+* access permission features
+* Speed of accessing and iterating through files (e.g. getting all files in a given directory)
+
+### Terminology:
+
+**Blob:** acronym for “Binary Large Object”
+
+## Considered Options
+
+* Blob Storage
+* File Services
+
+> Azure Files and Azure Blob Storage both offer ways to store large amounts of data in the cloud, but they are useful for slightly different purposes.
+>
+> Azure Blob Storage is useful for massive-scale, cloud-native applications that need to store unstructured data. To maximize performance and scale, Azure Blob Storage is a simpler storage abstraction than a true file system. You can access Azure Blob Storage only through REST-based client libraries (or directly through the REST-based protocol).
+>
+> Azure Files is specifically a file system. Azure Files has all the file abstracts that you know and love from years of working with on-premises operating systems. Like Azure Blob Storage, Azure Files offers a REST interface and REST-based client libraries. Unlike Azure Blob Storage, Azure Files offers SMB access to Azure file shares. By using SMB, you can mount an Azure file share directly on Windows, Linux, or macOS, either on-premises or in cloud VMs, without writing any code or attaching any special drivers to the file system. You also can cache Azure file shares on on-premises file servers by using Azure File Sync for quick access, close to where the data is used.
+
+### Option 1: Azure Blob Storage
+
+**Summary:** 
+Blob Storage is optimized for storing unstructured data: e.g. information that doesn't reside in a traditional row-column database.
```

Comment:
Maybe worthwhile outlining the different types of blob storage: 
`Block Blobs, Append Blobs and page blobs.`
https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs

and discussing which one we should use? Append blobs look interesting for the use case outlined here, add new approved transcriptions to the existing list.

Reply:
Very true, good idea. I will read up on the options and add my findings to the doc 👍